### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by kant**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,10 +32,10 @@ This scenario provides instructions for the following tasks:
 - Create local persistent volumes to define persistent disks.
 - Create a secret to protect sensitive data.
 - Create and deploy the WordPress frontend with one or more pods.
-- Create and deploy the MySQL database(either in a container or using Bluemix MySQL as backend).
+- Create and deploy the MySQL database (either in a container or using Bluemix MySQL as backend).
 
 ## Deploy to Bluemix
-If you want to deploy the wordpress directly to Bluemix, click on 'Deploy to Bluemix' button below to create a Bluemix DevOps service toolchain and pipeline for deploying the WordPress sample, else jump to [Steps](##steps)
+If you want to deploy the Wordpress directly to Bluemix, click on 'Deploy to Bluemix' button below to create a Bluemix DevOps service toolchain and pipeline for deploying the WordPress sample, else jump to [Steps](##steps)
```
Comment (loafyloaf) [inline README.md:38]: if we're getting specific, this should actually be 'WordPress'

File: README.md [hunk 2]
```diff
@@ -131,7 +131,7 @@ And the environment variables should look like this
           value: XMRXTOXTDWOOPXEE
 ```
 
-After you modified the `wordpress-deployment.yaml`, run the following commands to deploy wordpress.
+After you modified the `wordpress-deployment.yaml`, run the following commands to deploy Wordpress.
```
Comment (loafyloaf) [inline README.md:134]: ditto

File: README.md [hunk 3]
```diff
@@ -240,7 +240,7 @@ If you want to delete your persistent volume, you can run the following commands
 kubectl delete -f local-volumes.yaml
 ```
 
-If wordpress is taking a long time, you can debug it by inspecting the logs
+If Wordpress is taking a long time, you can debug it by inspecting the logs
 ```bash
 kubectl get pods # Get the name of the wordpress pod
```
Comment (loafyloaf) [inline README.md:245]: and once more

**Review by loafyloaf — CHANGES_REQUESTED** [review]
comments in-line

**Review by jkomg — APPROVED** [review]
LGTM

**Pull request closed without merging**